source/sass/components/article-page.scss

```diff
@@ -1,7 +1,5 @@
-/**
- * ArticlePage-specific styling.
- * Also see publication-page.scss and chapter-page.scss
- */
+// ArticlePage-specific styling.
```

Comment:
We have `postcss` set up to get rid of all comments in SCSS code in the compilation step. However, it's better that we stick to the `//` commenting style so comments don't get included without the help of `postcss`

See https://sass-lang.com/documentation/syntax/comments#in-scss